filtered-resource-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/filteredresource/kafka/KafkaRestWebSocketContractTest.java

```diff
@@ -272,9 +184,146 @@ void runContractTest(
         // Then
         // Assert each received response matches up with the expected
         assertThat(actualResponses)
+                .as("Testing Websocket Response")
                 .isEqualTo(websocketResponses);
+
+        if (auditErrorMessages.size() > 0) {
+
+            ConsumerSettings<String, AuditErrorMessage> consumerSettings = ConsumerSettings
+                    .create(akkaActorSystem, new StringDeserializer(), new ErrorDeserializer())
+                    .withBootstrapServers(KafkaInitializer.KAFKA_CONTAINER.getBootstrapServers())
+                    .withGroupId("error-topic-test-consumer")
+                    .withProperty(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+
+            Probe<ConsumerRecord<String, AuditErrorMessage>> errorProbe = Consumer
+                    .atMostOnceSource(consumerSettings, Subscriptions.topics(consumerTopicConfiguration.getTopics().get("error-topic").getName()))
+                    .runWith(TestSink.probe(akkaActorSystem), akkaMaterializer);
+
+
+            // When you read off the error queue
+            LinkedList<ConsumerRecord<String, AuditErrorMessage>> errorResults = LongStream.range(0, auditErrorMessages.size())
+                    .mapToObj(i -> errorProbe.requestNext(FiniteDuration.create(20 + auditErrorMessages.size(), TimeUnit.SECONDS)))
+                    .collect(Collectors.toCollection(LinkedList::new));
+
+            // Then
+            // The messages on the error topic are as expected
+            assertAll("Asserting on the error topic",
+                    () -> assertThat(errorResults)
+                            .as("Assert that there is one error on the error topic")
+                            .hasSize(1),
+
+                    () -> assertThat(errorResults.get(0).value())
+                            .as("Assert that after ignoring the Throwable object, and differences in timestamp, the AuditErrorMessages are the same")
+                            .usingRecursiveComparison()
+                            .ignoringFieldsOfTypes(Throwable.class)
+                            .ignoringFields("timestamp")
+                            .isEqualTo(auditErrorMessages.get(0)),
+
+                    () -> assertThat(errorResults.get(0).value().getError())
+                            .as("Assert that the error message inside the AuditErrorMessage is the same")
+                            .isExactlyInstanceOf(Throwable.class)
+                            .hasMessageContaining(auditErrorMessages.get(0).getError().getMessage())
+            );
+        }
     }
 
+    @ParameterizedTest
+    @ArgumentsSource(ParameterizedArgumentsNoResources.class)
+    @Disabled
```

Comment:
There is a ticket, PAL-971 for the reimplementation. In the interest of progress with examples, it was decided to mark this test as disabled so that the ticket could be finished, and then revisit it in the future